website/pages/community/index.jsx

```diff
@@ -5,39 +5,35 @@ import HashiHead from '@hashicorp/react-head'
 
 function CommunityPage() {
   return (
-    <div id="community">
+    <main id="community">
       <HashiHead is={Head} title="Community | Vault by HashiCorp" />
       <SectionHeader
         headline="Community"
         description="Vault is an open source project with a growing community. There are active, dedicated users willing to help you through various mediums."
-        use_h1={true}
+        useH1={true}
```

Comment:
Ouch 😬 This is a _super_ old prop from back in the Middleman / Preact days. Good catch.